packages/primitives/src/Badge/Badge.tsx

```diff
@@ -104,7 +104,7 @@ export const Badge = React.forwardRef(function Badge(
           position: hasLabel ? 'static' : 'absolute',
           py: 0,
           px: hasLabel ? 2 : '6px',
-          transform: '225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms',
+          transition: '225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms',
```

Comment:
Hmm I think we need to specify that the transition applies to either `all` or the `transform` but I don't see an animation when we do that 🤷‍♂️ .    If we need to show something without the upper badge, do we just render the component without the wrapper or is there a setting to hide the badge altogether?

Reply:
Ah yea I was missing the transform in the transition, thank you!

> If we need to show something without the upper badge, do we just render the component without the wrapper or is there a setting to hide the badge altogether?

What do you mean by this?

Reply:
I was just wondering how to render the element with no badge:

![image](https://user-images.githubusercontent.com/1864372/87816210-31512200-c824-11ea-92de-8af57064dde9.png)

here it's being shown by wrapping in a div but I was wondering if there was an option to pass in to hide the badge altogether.

Reply:
Oh hmm. Material UI has a `showZero` badge otherwise it automatically hides it when at zero.